apiserver/facades/agent/instancemutater/instancemutater.go

```diff
@@ -93,29 +94,36 @@ func NewInstanceMutaterAPI(st InstanceMutaterState,
 // CharmProfilingInfo returns info to update lxd profiles on the machine
 // based on the given unit names.  If the machine is not provisioned,
```

Comment:
"based on the given unit names." can be removed based on changes to CharmProfilingInfo()